snakesist/exist_client.py

```diff
@@ -21,6 +21,15 @@ class QueryResultItem(NamedTuple):
     node: NodeBase
 
 
+class ConnectionProps(NamedTuple):
+    root_collection: str
```

Comment:
what about sorting these as they would appear in a url? or alphabetically?

Reply:
good point, i'll sort them